Sources/Sentry/SentryTracer.m

```diff
@@ -917,20 +829,6 @@ + (void)resetAppStartMeasurementRead
     }
 }
 
-+ (nullable SentryTracer *)getTracer:(id<SentrySpan>)span
-{
-    if (span == nil) {
-        return nil;
-    }
-
-    if ([span isKindOfClass:[SentryTracer class]]) {
-        return span;
-    } else if ([span isKindOfClass:[SentrySpan class]]) {
-        return [(SentrySpan *)span tracer];
-    }
-    return nil;
-}
-
```

Comment:
Another part that I particularly disliked. The cast to SentrySpan, returned again as a SentryTracer, is really confusing.